jest.config.js

```diff
@@ -0,0 +1,37 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+    // Automatically clear mock calls and instances between every test
+    clearMocks: true,
+  
+    // An array of glob patterns indicating a set of files for which coverage information should be collected
+    collectCoverageFrom: ['<rootDir>/static/js/**/*.{js,jsx}'],
+  
+    // The directory where Jest should output its coverage files
+    coverageDirectory: 'coverage',
+  
+    // An array of file extensions your modules use
+    moduleFileExtensions: ['js', 'json', 'jsx'],
+  
+    // The paths to modules that run some code to configure or set up the testing environment before each test
+    setupFiles: ['<rootDir>/enzyme.config.js'],
+  
+    // The test environment that will be used for testing
+    testEnvironment: 'jsdom',
+  
+    // The glob patterns Jest uses to detect test files
+    testMatch: ['<rootDir>/static/**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).js?(x)'],
```

Comment:
I think it might be better to not have the `__tests__` directory here. Instead let's just keep the `.test.js` files next to the normal `js` files.